### DISCUSSION TITLE
Extracting LORA Error?

### DISCUSSION THREAD

**Discussion by le-khang** [hako-mikan/sd-webui-supermerger#120] [General]
Hi guys I got an error while trying to make LORA using the Super Merger extension. I tried different models and played around with the settings, but it just didn't give me any LORA. (I'm using Runpod with RTX 3090, this extension works well when I merge 2 Loras together)

Do you have any ideas on how to solve this error or any tips on how to extract LORA from my models? Thanks you very much.
![2023-06-22 17_33_55-Stable Diffusion](https://github.com/hako-mikan/sd-webui-supermerger/assets/35536989/c9b61f01-53ea-4c31-8a66-f99cbc302d3f)

**Comment by hako-mikan** [comment]
Maybe due to the version of diffusers.
https://github.com/hako-mikan/sd-webui-supermerger/discussions/52
If not, tell me errors on command prompt.

**Reply by le-khang** [reply]
Hi Hako Mikan thank you for your quick response. I'm not a technical person so I'm not sure how to "set the diffusers version to 0.10.2" as you mentioned in that thread. Could you please teach me how to do it? Additionally, I'm using the Runpod pre-made template, o it doesn't have command prompt (or perhaps it does, but I am unsure how to access it in Jupyter Lab).

**Reply by hako-mikan** [reply]
I don't know hot to do it.
Your environment is not same to my environment. So you cannot use my method.
As you read in https://github.com/hako-mikan/sd-webui-supermerger/discussions/52
`pip install diffusers==0.14.0` is the normal way in windows.
But i dont know the way for other environment,

**Comment by InResponse** [comment]
Closed discussion, I know, but it nonetheless seems the best place to ask: Is there an actual _reason_ to limit the diffusers version to 0.14.0? Like, does the script use a method that was deprecated in later versions or something?

**Reply by hako-mikan** [reply]
The subsequent update has made it possible for it to work with versions 0.14.0 and above of diffusers. 
There is no need to downgrade now.

**Reply by InResponse** [reply]
I was about to say, hurray, but upgrading to 0.20.1 from 0.14.0 breaks Lora extraction in an otherwise fresh venv. Downgrading fixes it again. Gives the same error as #125 .
